k2/csrc/context.h

```diff
@@ -321,12 +347,17 @@ ContextPtr GetCpuContext();
 // from PyTorch
 ContextPtr GetCudaContext(int32_t gpu_id = -1);
 
-// Returns a (CPU) context that will allocate pinned memory.  (This is CPU
-// memory that's pinned for faster GPU memory transfers).  May or may not
-// return the same value as ::k2::GetCpuContext()... this is so, for instance,
-// if you have a GPU PyTorch context you can get a CPU PyTorch context.
-// Returns a (CPU) context that will allocate pinned memory.
-ContextPtr GetPinnedContext();
+/* Returns a (CPU) context that will allocate pinned memory.  (This is CPU
+   memory that's pinned for faster GPU memory transfers).  May or may not
+   return the same value as ::k2::GetCpuContext()... this is so, for instance,
+   if you have a GPU PyTorch context you can get a CPU PyTorch context.
+   Returns a (CPU) context that will allocate pinned memory.
+
+   @param [in] device_type  If device_type is kCpu, it is equivalent
+                            to `GetCpuContext()`. If device_type is kCuda,
+                            it returns a context that allocates pinned memory.
```

Comment:
if `has_cuda` return false, it will return CPU context? maybe it's better to add documentation about this.

Reply:
Added.